python-avd/pyavd/_eos_designs/structured_config/base/monitor_sessions.py

```diff
@@ -77,13 +76,12 @@ def monitor_sessions(self: AvdStructuredConfigConnectedEndpointsProtocol) -> Non
 
             self.structured_config.monitor_sessions.append(monitor_session)
 
-    @cached_property
     def _monitor_session_configs(
```

Comment:
Why to remove cached_property? method is then ran twice for hosts with monitor session inputs.

```suggestion
    @cached_property
    def _monitor_session_configs(
```

Reply:
Yes, it is called twice. Fixed.